Doc/library/errno.rst

```diff
@@ -20,684 +20,436 @@ all-inclusive.
 
 To translate a numeric error code to an error message, use :func:`os.strerror`.
 
-Of the following list, symbols that are not used on the current platform are not
+Of the following table, symbols that are not used on the current platform are not
 defined by the module.  The specific list of defined symbols is available as
 ``errno.errorcode.keys()``.  Symbols available can include:
 
 
-.. data:: EPERM
+.. list-table::
+   :header-rows: 1
```

Comment:
See [`:widths:`](https://www.docutils.org/docs/ref/rst/directives.html#list-table)

This is relative column widths, so `1 1` would have the same effect. If you want the LHS column to be narrower then tweak the values

```suggestion
.. list-table::
   :header-rows: 1
   :widths: 50 50
```

Reply:
Hmm, it seems the problem is actually that the names in the middle and end of the table are quite long and don't wrap -- e.g. scroll to `errno.EPROTONOSUPPORT ` / `ESOCKTNOSUPPORT` / `ENOTRECOVERABLE`. Setting widths doesn't seem to help...